.eslintrc.js

```diff
@@ -27,23 +27,24 @@ module.exports = {
         'before': false, 'after': true
       }
     ],
-    'indent': [ 'error', 2 ],
+    'indent': ['error', 2],
     'keyword-spacing': [
       'error', { 'before': true }
     ],
-    'array-bracket-spacing': [ 'error', 'always' ],
+    'array-bracket-spacing': ['error', 'always'],
     'space-infix-ops': 'error',
-    'object-curly-spacing': [ 'error', 'always' ],
-    'semi': [ 'error', 'always' ],
-    'eol-last': [ 'error', 'always' ],
-    'comma-dangle': [ 'error', 'never' ],
+    'object-curly-spacing': ['error', 'always'],
+    'semi': ['error', 'always'],
+    'eol-last': ['error', 'always'],
+    'comma-dangle': ['error', 'never'],
     'no-multi-spaces': 'error',
     'no-multiple-empty-lines': 'error',
     'no-irregular-whitespace': 'error',
     'arrow-parens': ['error', 'always'],
-    'arrow-spacing': [ 'error', { before: true, after: true } ],
+    'arrow-spacing': ['error', { before: true, after: true }],
     'block-spacing': 'error',
-    'brace-style': [ 'error', '1tbs', { allowSingleLine: true } ],
-    'comma-style': 'error'
+    'brace-style': ['error', '1tbs', { allowSingleLine: true }],
+    'comma-style': 'error',
+    'max-len': ["error", { "code": 80, "tabWidth": 2, "comments": 65 }]
   }
 };
```

Comment:
These changed lines seem not related as the PR's title, may I ask you to revert these?